Scenarios/Scenario.py

```diff
@@ -0,0 +1,376 @@
+        :rtype: bool
+        """
+        logging.info("Start executing...")
+        try:
+            self.fleet.execute(clients=self.constellation)
+            logging.info("Finish executing...")
+            self.execution_success = True
+            return True
+        except RuntimeWarning as warning:
+            logging.info("Executing failed...")
+            self.execution_success = False
+            return warning
+
+    def define_constellation(self):
+        """ Based on input json, creates the orbits, 
+        the :class:`~Constellations.Constellation.Constellation` 
+        and fills the latter with :class:`~Spacecrafts.Sattelite.Sattelite` objects. 
+        """
+        # Define relevant orbits
+        logging.info("Gathering satellite orbits...")
+        self.define_constellation_orbits()
+
+        # Define a reference satellite
+        logging.info("Generating the reference satellite...")
+        self.reference_satellite = Satellite('Reference_' + self.constellation_name + '_satellite',
+                                             self.sat_mass,
+                                             self.sat_volume,
+                                             insertion_orbit=self.sat_insertion_orbit,   
+                                             operational_orbit=self.sat_operational_orbit,
+                                             disposal_orbit=self.sat_disposal_orbit,
+                                             state='standby',
+                                             default_orbit=self.sat_default_orbit,
+                                             is_stackable=False)
+
+        # Instanciate ConstellationSatellites object
+        logging.info("Instanciating the constellation...")
+        self.constellation = Constellation(self.constellation_name)
+
+        logging.info("Populating the constellation based on the reference satellite...")
+        self.constellation.populate_standard_constellation(self.constellation_name,
+                                                           self.reference_satellite,
+                                                           number_of_planes=self.n_planes,
+                                                           sat_per_plane=self.n_sats_per_plane,
+                                                           plane_distribution_angle=self.plane_distribution_angle,
+                                                           altitude_offset=0 * u.km)
+
+        # Log satellites distribution
+        for _, satellite in self.constellation.satellites.items():
+            logging.info(f"Sat {satellite.get_id()} has {satellite.get_default_orbit()}, {satellite.get_default_orbit().raan} RAAN, {satellite.get_default_orbit().nu} nu orbit")
+
+        # Plot if verbose
+        if self.verbose:
+            logging.info("Start plotting Clients...")
+            self.satellites.plot_3D_distribution(save="3D_plot", save_folder=self.data_path)
```

Comment:
@malogoury-tcdc @EmiTCDC @MatUspace 
When verbose is active I get an attribute error here: `AttributeError: 'ScenarioConstellation' object has no attribute 'satellites'`. What's the updated name for the `satellites` attribute?